.changeset/light-horses-hammer.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/theme': patch
+---
+
+Replace the link color in dark theme to #FDFB9F instead dark blue
```

Comment:
```suggestion
Replace the link color in dark theme
```

...suggest you don't need to specify what color it is, since those reviewing the CHANGELOG won't be able to tell what that hex code is anyway.  Instead, if interested they can simply open the change via this PR to see exactly what color you DID choose. Key point for a change record is that you changed the color, and it's for the dark theme, if that makes sense :)